scripts/docker-compose.yml

```diff
@@ -1,21 +1,21 @@
 version: '3.7'
 services:
   artifactory-1:
-    image: releases-docker.jfrog.io/jfrog/artifactory-pro:7.27.10
+    image: releases-docker.jfrog.io/jfrog/artifactory-pro:${ARTIFACTORY_VERSION}
```

Comment:
you can actually do `${ARTIFACTORY_VERSION:-7.2.23}` which will default it if not present.

Reply:
True but I don't to have 2 default values in here and the shell script.